bundle/ScheduledVisibility/Registry.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility;
+
+use OutOfBoundsException;
+
+use function sprintf;
+
+final class Registry
+{
+    /**
+     * @var ScheduledVisibilityInterface[]
```

Comment:
All PHPDoc should use the FQCN.

Reply:
fixed in 4afa970a8032e6d05cee1a491cc7f1f26660782e